platform/darwin/src/MGLSymbolStyleLayer.h

```diff
@@ -150,6 +150,24 @@ typedef NS_ENUM(NSUInteger, MGLSymbolPlacement) {
     MGLSymbolPlacementLineCenter,
 };
 
+/**
+ Controls the order in which overlapping symbols in the same layer are rendered
+
+ Values of this type are used in the `MGLSymbolStyleLayer.symbolZOrder`
+ property.
+ */
+typedef NS_ENUM(NSUInteger, MGLSymbolZOrder) {
+    /**
+     Symbols will be sorted by their y-position relative to the viewport.
+     */
+    MGLSymbolZOrderViewportY,
+    /**
+     Symbols will be rendered in the same order as the source data with no
+     sorting applied.
+     */
+    MGLSymbolZOrderSource,
```

Comment:
So I tried overwriting the enum names and descriptions in style-spec-cocoa-conventions-v8.json but instead of overwriting the existing enums, it simply added two new ones. In other words, there would now be `MGLSymbolZOrderSource` and `MGLSymbolZOrderBySource` so I'm not sure this is possible with the current setup, unless I'm missing something.

Reply:
Huh, looks like you’re right. I guess it isn’t worth building out the capability to override individual enumeration values, given that the literal string values are more common and can’t change anyways (since they can be built up through concatenation etc.).

/cc @friedbunny